<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>438</width>
    <height>404</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>NotePoster</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Notebook:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="notebooksComboBox"/>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLineEdit" name="titleLineEdit">
     <property name="placeholderText">
      <string>Title</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QTextEdit" name="noteTextEdit"/>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QToolButton" name="boldButton">
         <property name="text">
          <string>Bold</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="italicButton">
         <property name="text">
          <string>Italic</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="sendButton">
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>Send!</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>boldButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setBold(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>397</x>
     <y>75</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>72</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>italicButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>MainWindow</receiver>
   <slot>setItalic(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>393</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>429</x>
     <y>97</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>sendButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>createNote()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>381</y>
    </hint>
    <hint type="destinationlabel">
     <x>433</x>
     <y>380</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>setBold(bool)</slot>
  <slot>setItalic(bool)</slot>
  <slot>createNote()</slot>
 </slots>
</ui>
